casadm/statistics_model.c

```diff
@@ -528,10 +528,10 @@ int cache_stats_conf(int ctrl_fd, const struct kcas_cache_info *cache_info,
 	char dev_path[MAX_STR_LEN];
 	int inactive_cores;
 
-	if (get_dev_path(cache_info->cache_path_name, dev_path, sizeof(dev_path)) != SUCCESS)
-		cache_path = cache_info->cache_path_name;
-	else
+	if (!full_path && get_dev_path(cache_info->cache_path_name, dev_path, sizeof(dev_path)) == SUCCESS)
 		cache_path = dev_path;
```

Comment:
it looks wrong. If get_dev_path returned SUCCESS then we would like to use cache_info->cache_path_name

Reply:
I can't agree. `get_dev_path` is an old function (I didn't modify it) that sets the value of the first argument to the second. If it succeeds (and we don't want full path) we want to set `cache_path` to `dev_path` which is a short path now. Otherwise set `cache_info->cache_path_name` as `cache_path`.